### DISCUSSION TITLE
MySQL with Deta

### DISCUSSION THREAD

**Discussion by MikaRinquest** [deta/cloud-docs#720] [Help]
It connected to mysql since I can use a get method, but nothing happens when I use a post method. Any ideas on how to fix or should I just use the built in database?

**Comment by jnsougata** [comment]
we need more than this to look into your issue! please provide more info.

**Reply by MikaRinquest** [reply]
I tried connecting mysql to deta, and it looked like it was working because when I tried retrieving information it worked, but whenever I tried sending information though it wouldn't. Can deta connect to MySQL, because I couldn't find anything from videos or the documentation.
